src/ts/component/block/text.tsx

```diff
@@ -5,24 +5,19 @@ import $ from 'jquery';
 import raf from 'raf';
 import { observer, } from 'mobx-react';
 import { Select, Marker, Loader, IconObject, Icon, Editable } from 'Component';
-import { I, C, S, U, J, keyboard, Key, Preview, Mark, focus, Storage, translate, analytics, Renderer } from 'Lib';
+import {
+	I, C, S, U, J, keyboard, Key, Preview, Mark, focus, Storage,
+	translate, analytics, Renderer,
+	getPrismComponents, prismLanguagesAndAliases,
```

Comment:
This should not be separate classes in Lib, it should be somewhere in common, menu or data.